src/main/resources/config/systemMG1Preemp/SPECIFIC_VALIDATOR_CONSTRAINTS.json

```diff
@@ -7,7 +7,8 @@
   "values": [
     {
       "value": [
-        "smallerLambdaEsMG1Validator"
+        "smallerLambdaEsMG1Validator",
+        "MG1PriorThirdClassValidator"
```

Comment:
Shouldn't we use lowercase for the naming? The bean name will be lowercase for sure.

Reply:
Discussed.